packages/sdk/src/oracle/utils.ts

```diff
@@ -189,3 +191,10 @@ export class TransactionConfirmer {
     return false;
   }
 }
+
+export function chainConfigToChainMetadata(config: ChainConfig): ChainMetadata {
+  // linters hate this, but we just need to strip out these vars for valid chain metadata
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
You could also add this option to the eslint config so this doesn't cause issues here or in the future!
https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings

Reply:
i generally do like this rule!

Reply:
oh thanks for correcting me, this rule is specificallly for this case, i will add that